### PR TITLE
Us 10513 - Cookie page 

### PR DESCRIPTION
**Pull request by AkhilaHMRC**

### CODE REVIEW

**Review by monalisasarkar123 — APPROVED** [review]

**Review by SushreeM-h — APPROVED** [review]

File: src/samples/ChildBenefitsClaim/CookiePageTable.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```
Comment (AkhilaHMRC) [inline src/samples/ChildBenefitsClaim/CookiePageTable.tsx:1]: Since the cookie page is a static page, you can add this table in cookie page. Anyhow this table is not reusable. or create a component folder 'cookiePage' and add both the components in there

File: src/components/AppComponents/AppHeader/index.tsx [hunk 2]
```diff
@@ -24,14 +24,15 @@ export default function AppHeader(props) {
             </a>
           </div>
           <div className="govuk-header__content">
-            <a href="#" className="hmrc-header__service-name hmrc-header__service-name--linked" id="service-header">
+            <a href="/" className="hmrc-header__service-name hmrc-header__service-name--linked" id="service-header">
```
Comment (AkhilaHMRC) [inline src/components/AppComponents/AppHeader/index.tsx:27]: undo this change, this will create issue in higher env, where we have a path for our application

File: src/i18n.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ i18n
     lng: sessionStorage.getItem('rsdk_locale')?.substring(0, 2) || 'en',
     backend: {
       /* translation file path */
-      loadPath: `${window.location.pathname}assets/i18n/{{lng}}.json`
+      loadPath: `${window.location.pathname.replace('/cookies','')}assets/i18n/{{lng}}.json`
```
Comment (AkhilaHMRC) [inline src/i18n.js:12]: Do we need this script, since you have moved the init to appSelector ?

Comment (StefanCogn) [reply src/i18n.js]: No I guess not - refactored here [e312177](https://github.com/norm-l/odx/pull/123/commits/e312177a419463cdc4dbb3759f01a8bebda331e7)

File: src/components/AppComponents/AppHeader/index.tsx (hunk 2 above)
Comment (StefanCogn) [reply src/components/AppComponents/AppHeader/index.tsx]: Thanks, resolved here [e312177](https://github.com/norm-l/odx/pull/123/commits/e312177a419463cdc4dbb3759f01a8bebda331e7)

File: src/samples/ChildBenefitsClaim/CookiePageTable.tsx (hunk 1 above)
Comment (StefanCogn) [reply src/samples/ChildBenefitsClaim/CookiePageTable.tsx]: Thanks, resolved here [e312177](https://github.com/norm-l/odx/pull/123/commits/e312177a419463cdc4dbb3759f01a8bebda331e7)

**Review by monalisasarkar123 — CHANGES_REQUESTED** [review]
Please resolve the conflicts in this PR

**Comment by StefanCogn** [comment]
> Please resolve the conflicts in this PR

Thanks, addressed here [99c453a](https://github.com/norm-l/odx/pull/123/commits/99c453ad6f6b71bb75662bd7f2fc1c88ca9ae53a)

**Review by monalisasarkar123 — APPROVED** [review]

**Review by StefanCogn — APPROVED** [review]
